recipes/r-rockchalk/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = '1.8.144' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rockchalk
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rockchalk_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rockchalk/rockchalk_{{ version }}.tar.gz
+  sha256: cb5e6c87ec95589f2b02da7ed7a5a67ec125e132100c8ae6762fad52820db71d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-mass
+    - r-cardata
+    - r-kutils
+    - r-lme4
+
+  run:
+    - r-base
+    - r-mass
+    - r-cardata
+    - r-kutils
+    - r-lme4
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('rockchalk')"           # [not win]
+    - "\"%R%\" -e \"library('rockchalk')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=rockchalk
+  license: GPL-3.0-or-later
+  summary: A collection of functions for interpretation and presentation of regression analysis.  These
+    functions are used to produce the statistics lectures in <http://pj.freefaculty.org/guides>.
+    Includes regression diagnostics, regression tables, and plots of interactions and
+    "moderator" variables. The emphasis is on "mean-centered" and "residual-centered"
+    predictors. The vignette 'rockchalk' offers a fairly comprehensive overview.  The
+    vignette 'Rstyle' has advice about coding in R.  The package title 'rockchalk' refers
+    to our school motto, 'Rock Chalk Jayhawk, Go K.U.'.
+
+  license_family: GPL3
+  license_file:
+    - LICENSE.txt
```

Comment:
```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

Reply:
Please remove the `license.txt` file and instead reference the license file that `r-base` already installs.